integration_tests/src/main/python/asserts.py

```diff
@@ -23,6 +23,8 @@
 import time
 import types as pytypes
 import data_gen
+import difflib
```

Comment:
Do we need to add difflib as a dependency in the docs?

Reply:
Not sure if it is really necessary since it's a built-in thing https://docs.python.org/3.10/library/difflib.html

Reply:
btw diff is a separate PR, #8918, which I hope can be merged soon